<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="del1-lhp-n78153" failures="1" tests="4" name="Login Page Test" time="42.016" errors="0" timestamp="2023-09-18T13:48:20 IST">
  <testcase classname="com.qa.opencart.tests.LoginPageTest" name="loginPageSpecificDataTest" time="7.103">
    <skipped/>
  </testcase> <!-- loginPageSpecificDataTest -->
  <testcase classname="com.qa.opencart.tests.LoginPageTest" name="loginPageSpecificDataTest" time="6.044">
    <skipped/>
  </testcase> <!-- loginPageSpecificDataTest -->
  <testcase classname="com.qa.opencart.tests.LoginPageTest" name="loginPageSpecificDataTest" time="6.217">
    <failure type="java.lang.AssertionError" message="expected [Fail] but found [Please make sure you&amp;apos;ve entered the correct email address and password and try again. If you&amp;apos;ve forgotten your password, we canhelp you recover it]">
      <![CDATA[java.lang.AssertionError: expected [Fail] but found [Please make sure you've entered the correct email address and password and try again. If you've forgotten your password, we canhelp you recover it]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.qa.opencart.tests.LoginPageTest.loginPageSpecificDataTest(LoginPageTest.java:38)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:204)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:443)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- loginPageSpecificDataTest -->
  <testcase classname="com.qa.opencart.tests.LoginPageTest" name="loginPageSpecificDataTest2" time="6.108"/>
</testsuite> <!-- Login Page Test -->
